### ISSUE TITLE
Maximum length of strings in Models and Actions

### ISSUE THREAD

**Issue by ClFeSc** [hpi-sam/digital-fuesim-manv#232]
We should validate the maximum string length.

**Comment by christianzoellner** [comment]
easy but monotonous work.

make sure to get _all_ form fields.

**Comment by ClFeSc** [comment]
Note that this is primarily about the models and actions in `shared` and not focused on the forms in the UI.

**Comment by Dassderdie** [comment]
Should be done after #125

**Comment by lukasrad02** [comment]
Makes sense to build upon angular forms with Zod schema validation, once actions are validated using Zod instead of class-validator
